network-api/networkapi/mozfest/templates/partials/primary_heroguts.html

```diff
@@ -25,9 +25,9 @@ <h1 class="h2-heading mb-0">{% if page.header %}{{ page.header }}{% else %}{{ pa
   <div class="container">
     <div class="row no-gutters">
       <div class="cutout pt-4 col-12 {% if homepage != True %}col-md-8{% endif %}">
-        <h1 class="h2-heading mb-0">{% if root.title %}{{ root.title }}{% elif page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}</h1>
+        <h1 class="h1-heading mb-0">{% if root.title %}{{ root.title }}{% elif page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}</h1>
         {% if page.intro %}
-        <p class="body-large mt-3 mb-0">{{ page.intro | richtext }}</p>
+        <div class="intro body-large mt-3">{{ page.intro | richtext }}</div>
```

Comment:
`{{ page.intro | richtext }}` returns block-level markup (e.g., `<p>`, `<ul>` ) and hence can't be wrapped within  the `<p>` wrapper

Reply:
oh dang, good catch!